-1 
Tested at: c879ba21a8b788d06b6fd765c53edb346bcd490c
 I found an error when building: 
 
 <pre>gmake[1]: *** [CompilePython] Error 1
gmake[1]: Target `PostBuild' not remade because of errors.
gmake[1]: Leaving directory `/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-02-1100'
gmake: *** [src] Error 2
gmake: Target `all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8646/4076/summary.html

--- a/Configuration/HLT/python/autoHLT.py
+++ b/Configuration/HLT/python/autoHLT.py
@@ -3,9 +3,9 @@
 
 autoHLT = {
   'frozen25ns' : '25ns14e33_v1',
-  'relval25ns' : 'GRun'
+  'relval25ns' : 'GRun',
   'frozen50ns' : '50ns_5e33_v1',
-  'relval50ns' : '50nsGRun'
+  'relval50ns' : '50nsGRun',
   'fake'       : 'Fake',
   'test'       : 'GRun',
 }

